modules/claims_api/app/swagger/claims_api/forms/form_2122_response_swagger.rb

```diff
@@ -220,7 +220,7 @@ class Form2122ResponseSwagger
                 key :type, :object
                 property :veteran do
                   key :type, :string
-                  key :description, 'Base64 encoded png image of the veterans signature'
+                  key :description, 'Base64 encoded png image of the veteran or claimant signature'
```

Comment:
This isn't the file where ya wanna make this update.
If you notice in the `swagger.json` file, you'll see that your verbiage update isn't reflected in the new version of the `swagger.json` file.

You want to make this change [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/config/schemas/v2/2122.json#L347).

Not sure if you know how to do it yet, but you can also run the `developer-portal` locally so you can see how the updated`swagger.json` file will effect the developer docs.  It's a good way of making sure the changes you want to make to the dev docs look the way you want.  

Just let me know if you want me to show you how to do that.